functions/commonfn.php

```diff
@@ -357,6 +361,33 @@ function amuRegisterUser(){
 		//return message
 		return $newid;
 	}
+
+    /**
+     * Contacts central auth to determine if uun exists
+     *
+     * @param string $uun
+     *
+     * @return boolean
+     */
+    function is_valid_uun( $uun ) {
+        global $amu_ldap_config;
```

Comment:
Is there any way we can avoid this global? No worries if there isn't but wanted to check with you as would be good to avoid adding further globals if possible. Thanks

Reply:
I have added the global to the constructor(and assignment there), but we would need to use a global elsewhere if we moved it out of the class. think this is probably the best place for it, but not in the function.

Reply:
Thanks Richard.